index.md

```diff
@@ -6,18 +6,23 @@ use-site-title: true
 ---
 
 This is the story of how three Software Engineers decided to shed some light on Background Job Processors by building one from scratch.
-We started by understanding the use for a Background Job Processor (BJP), then, identifying the most popular ones in the Ruby-ecosystem and, finally, building one from scratch.
-In this post we will walk you through the process of building a BJP from scratch and how we managed to build one, **Workerholic**, that compares to Sidekiq in terms of performance.
+We started by acquiring a deeper understanding about the use for a Background Job Processor (BJP), then, we identified the most popular ones in the Ruby-ecosystem and, finally, we built one from scratch.
```

Comment:
`We started by getting a deeper understanding of the use-cases for Background Job Processor (BJP), then we identified the most popular BJPs in the Ruby-ecosystem and, finally, we built one from scratch.`